File: challenges/day05/fizzbuzz.c
#include <stdio.h>
#include <stdlib.h>

int main() {
	for (int i = 1; i <= 100; ++i)
	{
		if(i % 15 == 0) printf("%d Fizzbuzz\n",i);
		else if( i % 3 == 0 ) printf("%d Fizz\n",i);
		else if( i % 5 == 0 ) printf("%d Buzz\n",i);
		else printf("%d\n",i);
	}
	return EXIT_SUCCESS;
}

File: challenge/day05/fizzbuzz.php
<?php
/**
 * @author:Raju Lamsal
 * @copyright 2019
 * 
*/ 
for($i=1;$i<=100;$i++){
    if($i%15==0 and $i%5==0){
        echo "<br/> Fizzbuzz";
    }elseif($i%3==0){
        echo "<br/> Fizz";
    }elseif($i%5==0){
        echo "<br/> Buzz";
    }else{
        echo "<br/>" . $i;
    }
}
?>